assets/src/components/parts/janus-capi-iframe/CapiIframeAuthor.tsx

```diff
@@ -163,43 +243,102 @@ const CapiIframeAuthor: React.FC<AuthorPartComponentProps<CapiIframeModel>> = (p
 
     // Introducing listeners requires returning a function that also un-listens
     return () => {
-      /* console.log('%c MESSAGE LISTENER UNLOADED', 'background: darkred; color: #fff;', {
-        activityId: id,
-      }); */
       // unlisten to post message calls
       window.removeEventListener('message', messageListener.current);
     };
   }, [simFrame]);
+  /**
+   * Performs a deep copy, or clone, of an object.
+   *
+   * @param o the object to clone
+   * @returns the cloned object
+   */
+  const clone = (o: any) => {
```

Comment:
why not importing this?

Reply:
Fixed. I thought that clone function was inside the textflow component and hence I added new. I didn't knew it was it util/common